tests/resource/test_measurement.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2018-2022 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Unit tests for functions needed for estimating complexity of measuring expectation values.
+"""
+import pytest
```

Comment:
The one mistake that I can see users making with this function is to pass the wrong format for the coefficients. I wonder if we can test this or raise a warning? Just something to consider, optionally